### DISCUSSION TITLE
Missing Competition Section in Docs?

### DISCUSSION THREAD

**Discussion by rsaccone** [hmlongco/Factory#104] [Q&A]
Hello,

A little while back you had started a write up comparing the pros and cons various alternative libraries to Factory. I had a link to that section of the docs but it doesn’t work anymore.  Curious if you’ve abandoned the comparison work or if you are doing it in another venue?

Regards,
Rob

**Comment by hmlongco** [comment]
I basically pulled it until I had the time to do a more thorough job researching the differences.

There is a document for "Design" that explains some of the rationale behind Factory and Factory 2.1, if you're interested.

**Reply by NitishGadangi** [reply]
@hmlongco Will be really helpful. Can you share the link here.

**Reply by hmlongco** [reply]
It's not online and I'm not sure when or if I'm going to have the time to complete it.
